src/blueapi/worker/reworker.py

```diff
@@ -193,11 +215,11 @@ def _report_status(
         warnings = self._warnings
         if self._current is not None:
             task_status = TaskStatus(
-                task_name=self._current.name,
+                task_name=self._current.task_id,
                 task_complete=self._current.is_complete,
                 task_failed=self._current.is_error or bool(errors),
             )
-            correlation_id = self._current.name
+            correlation_id = self._current.task_id
```

Comment:
Wasn't the correlationId going to be different to the taskId? Doesn't it have to be provided by the client?

Reply:
Yes, but we're not doing anything to do with the correlation ID in this change, so preserving the old behaviour (where it is the same) for now.

Reply:
@keithralphs agreed but still don't want to bloat this change any further. Linking to #177 for reference so this gets done